actors/power/tests/power_actor_tests.rs

```diff
@@ -198,3 +201,128 @@ fn claimed_power_given_claim_does_not_exist_should_fail() {
     rt.verify();
     h.check_state();
 }
+
+const MINER1: Address = Address::new_id(111);
+const MINER2: Address = Address::new_id(112);
+const MINER3: Address = Address::new_id(113);
+const MINER4: Address = Address::new_id(114);
+const MINER5: Address = Address::new_id(115);
+
+#[test]
+fn power_and_ledge_accounted_below_threshold() {
+    let small_power_unit = &StoragePower::from(1_000_000);
+    let small_power_unit_x2 = &(small_power_unit * 2);
+    let small_power_unit_x3 = &(small_power_unit * 3);
+
+    let (mut h, mut rt) = setup();
+
+    h.create_miner_basic(&mut rt, *OWNER, *OWNER, MINER1).unwrap();
+    h.create_miner_basic(&mut rt, *OWNER, *OWNER, MINER2).unwrap();
```

Comment:
So that this test is robust against changes to network params assert that the min consensus miners count is > 2